src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -550,7 +550,6 @@ static smt_termt convert_expr_to_smt(const index_exprt &index)
 
 static smt_termt convert_expr_to_smt(const shift_exprt &shift)
 {
-  // TODO: Dispatch into different types of shifting
```

Comment:
This should not be completely removed, as the dispatching is not complete. The implementation of `can_cast_expr<shift_exprt>` supports casting rotations to `shift_exprt`. So the dispatching is not done until we account for the rotation expressions.

Reply:
This change has been reversed.